### DISCUSSION TITLE
Top Langs Not Working

### DISCUSSION THREAD

**Discussion by AiltonDevesse** [anuraghazra/github-readme-stats#1717] [Q&A] [answered]
Hello,

For some reason the top langs API doesn't work for my user, i tried: 
https://github-readme-stats.vercel.app/api/top-langs/?username=ailtondevesse
![image](https://user-images.githubusercontent.com/28234293/164976632-69262107-9ccc-4a12-9602-d7b96a86114e.png)

**Comment by rickstaa** [comment, marked as answer]
@AiltonDevesse Is this still an issue? I just checked your username, and the language card is working for me (see https://github-readme-stats.vercel.app/api/top-langs/?username=ailtondevesse). Please be aware that we use cache for the cards to reduce the server load (see https://github.com/anuraghazra/github-readme-stats#common-options).

**Comment by AiltonDevesse** [comment]
Hello,

After some time it started working,
Thank you

—
Kind regards
Ailton Devesse
